wn/web.py

```diff
@@ -320,7 +321,22 @@ async def synset(request):
     return JSONResponse({'data': make_synset(synset, request)})
 
 
+async def index(request: Request):
+    global routes
+    endpoints = [{ 'path': route.path,'name': route.name} for route in routes]
+    return JSONResponse({'endpoints': endpoints})
+
+
+async def health_check(request: Request):
+    return JSONResponse({
+        "status": "healthy",
+        "timestamp": datetime.now(tz=UTC).isoformat(),
+        "service": "wordnet",
+    }, status_code=200)
```

Comment:
I couldn't find any standard or strong convention for a health or status check except this expired proposal: https://inadarei.github.io/rfc-healthcheck/

I don't think there's a need to conform to that proposal, so I mostly just have code style suggestions, but I'd also change the service name to `wn.web`:

```suggestion
    body = {
        'status': 'healthy',
        'timestamp': datetime.now(tz=UTC).isoformat(),
        'service': 'wn.web',
    }
    return JSONResponse(body, status_code=200)
```